src/current/_includes/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,178 @@
+These essential CockroachDB metrics enable you to build custom dashboards with tools, such as: 
+{% if include.deployment == 'self-hosted' %}
+* [Grafana](monitor-cockroachdb-with-prometheus.html#step-5-visualize-metrics-in-grafana)
+* [Datadog Integration](datadog.html) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics) column lists the corresponding Datadog metric which requires the `cockroachdb.` prefix. A given metric is not available using the Datadog Integration if the **Datadog Integration Metric Name** column is blank (such as for the `sys.cpu.host.combined.percent-normalized` metric).
```

Comment:
I think it might make more sense to write "not available" or "N/A" in the blank columns instead of having this info here

Reply:
Updated